src/commands/domains/add.ts

```diff
@@ -0,0 +1,73 @@
+import { Command, flags } from '@oclif/command'
+
+import * as api from '../../api'
+import setupApiClient from '../../setupApiClient'
+
+import { handle } from 'oazapfts'
+import withStandardErrors from '../../utils/errorHandling'
+
+type FlagsType = {
+  domain?: string
+
+  [key: string]: any
+}
+
+export default class DomainsAdd extends Command {
+  static description = 'add a domain'
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    domain: flags.string({
+      char: 'd',
+      description: 'the domain to add to your account',
```

Comment:
Capitalize the. Double check all areas which may need capitalization.

Reply:
Sticking with the rest of the help command description formatting

Reply:
Oh I was unaware that is formatted that way.